gradle.properties

```diff
@@ -20,6 +20,10 @@ csdePluginVersion=1.2.0-beta-+
 # Specify the name of the workflows module
 workflowsModule=workflows
 
+# Metadata for the CorDapp.
+cordappLicense="Apache License, Version 2.0"
+cordappVendorName="R3"
```

Comment:
Added as most CorDapp examples I've come across have a license specified. The specific license version can be removed and set to a blank string